[CRUD] Frozen Node Admin

tpl2go opened the pull request:
## Log
- Installed CRUD interface [FrozenNode/Laravel-Administrator](https://github.com/FrozenNode/Laravel-Administrator/). 
- Applied patch https://github.com/FrozenNode/Laravel-Administrator/pull/973
## TODO
- [x] input fields of `config/administrator/members.php`
- [ ] ~~add authentication to protect~~ (deferred to another PR)
- [x] foreign key in database
- [x] add patch
- [ ] ~~figure out why view is so long horizontally~~ (deferred)
## Test Plan

View on [http://localhost:8000/admin/members](http://localhost:8000/admin/members)

tpl2go committed:
[CRUD] Frozen Node Admin

wonglkd commented:
Note: this PR involved the application of patch found in https://github.com/FrozenNode/Laravel-Administrator/pull/973 .

tpl2go committed:
added column names

tpl2go committed:
added colleges, courses, scholarships table

tpl2go committed:
removed public/package; added course_others

tpl2go committed:
updated .travis/deploy.sh;

tpl2go committed:
added composer.lock to .gitignore

tpl2go committed:
rm composer.lock

tpl2go committed:
added course_others

tpl2go committed:
added composer.lock

wonglkd committed:
Make composer get forked version of FrozenNode Administrator

wonglkd committed:
Cleanup FrozenNode admin console to make it usable, fix relations in Eloquent models, reinstate auth check for FrozenNode CRUD panel

wonglkd committed:
Merge branch 'master' into tpl_crud

wonglkd committed:
Fixed authentication check in case of null (as per Spencer's comment)
http://laravel.io/forum/09-15-2015-laravel-5-can-not-get-authenticated-user-from-view

wonglkd commented on config/administrator.php:
```diff
@@ -0,0 +1,149 @@
+	 */
+	'permission'=> function()
+	{
+		return Auth::check() && Auth::user()->hasRole('editor');
```
For reference

> Spencer: did you check if Auth::user() is null if not authenticated
> http://laravel.io/forum/09-15-2015-laravel-5-can-not-get-authenticated-user-from-view

(This originally omitted the `Auth::check()`.)
